src/_pytest/terminal.py

```diff
@@ -474,10 +473,7 @@ def pytest_runtest_logreport(self, report: TestReport) -> None:
             else:
                 markup = {}
         if self.verbosity <= 0:
-            if not running_xdist and self.showfspath:
-                self.write_fspath_result(rep.nodeid, letter, **markup)
```

Comment:
This is never reached?

Reply:
It is reached, but has the same outcome as just the `write`.

What `write_fspath_result` does is check whether the path (extracted from the nodeid) is different than the last call to `write_fspath_result`, and if so, starts a new line and prints the path, before printing the argument.

The reason it's not needed here is that `pytest_runtest_logstart` already calls `write_fspath_result`, so the call here (`logreport`) doesn't.

Because the check it performs is mildly expensive, it's good to avoid it when possible.